src/guiapi/src/MItem_menus.cpp

```diff
@@ -218,3 +220,38 @@ MI_FOOTER_SETTINGS::MI_FOOTER_SETTINGS()
 void MI_FOOTER_SETTINGS::click(IWindowMenu & /*window_menu*/) {
     Screens::Access()->Open(GetScreenMenuFooterSettings);
 }
+
+MI_LANGUAGUE_USB::MI_LANGUAGUE_USB()
+    : WI_LABEL_t(_(label), 0, is_enabled_t::yes, is_hidden_t::no) {}
+
+void MI_LANGUAGUE_USB::click(IWindowMenu &windowMenu) {
+    if (fileProviderUSB.OpenFile())
+        ProviderRegistrator("ts", &fileProviderUSB);
+}
+
+MI_LOAD_LANG::MI_LOAD_LANG()
+    : WI_LABEL_t(_(label), 0, is_enabled_t::yes, is_hidden_t::no) {}
+
+void MI_LOAD_LANG::click(IWindowMenu &windowMenu) {
+
+    uint8_t buff[512];
```

Comment:
please define a symbolic constant for buffer length and put 16 here ... allocating 512B on the stack may end up with stack overflow